src/app/api/_api-services/redis_service/index.ts

```
@@ -618,4 +630,72 @@ export class RedisService {
+		proposalType,
+		indexOrHash,
+		votesType,
+		analyticsType
+	}: {
+		network: string;
+		proposalType: string;
+		indexOrHash: string;
+		votesType: EPostTilesVotesType;
+		analyticsType?: EAnalyticsType;
+	}): Promise<IPostTilesVotes | null> {
+		const data = await this.Get({ key: this.redisKeysMap[ERedisKeys.POST_TILES_VOTES_DATA](network, proposalType, indexOrHash, votesType, analyticsType) });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)